src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/cancel.html.twig

```diff
@@ -0,0 +1,25 @@
+{% block toolbar %}
+    {% set icon %}
+        {{ include('@WebProfiler/Icon/symfony.svg') }}
+
+        <span class="sf-toolbar-value sf-toolbar-ajax-request-counter">
+            Loading&hellip;
+        </span>
+    {% endset %}
+
+    {% set text %}
+        <div class="sf-toolbar-info-piece">
+            <b>Loading the web debug toolbar&hellip;</b>
+        </div>
+        <div class="sf-toolbar-info-piece">
+            Attempt #<span id="sfLoadCounter-{{ token }}"></span>
+        </div>
+        <div class="sf-toolbar-info-piece">
+            <b>
+                <a id="sfLoadCancel-{{ token }}" href="#">Cancel</a>
```

Comment:
should probably be a `<button type="button">`, for better accessibility (it behaves like a button, not like a link, so it should be a button)

Reply:
done